apheleia-formatters.el

```diff
@@ -47,7 +47,7 @@
                   (apheleia-formatters-indent
                    "--indent-with-tabs" "--indent-size"))
     (dart-format . ("dart" "format"))
-    (dprint . ("dprint" "fmt" "--stdin" (file-name-nondirectory filepath)))
+    (dprint . ("dprint" "fmt" "--stdin" filepath))
```

Comment:
As discussed - it should be fine to give the full `filepath`, with the other fixes below.